spec/lib/authorities/linked_data/term_config_spec.rb

```diff
@@ -8,14 +8,14 @@
   let(:encoding_config) { Qa::Authorities::LinkedData::Config.new(:LOD_ENCODING_CONFIG).term }
   let(:loc_config) { Qa::Authorities::LinkedData::Config.new(:LOC).term }
 
-  let(:ldpath_results_config) do
+  let(:predicate_results_config) do
     {
```

Comment:
This is the primary change for the PR. The config specified results as ldpaths instead of predicates.